third_party/tarantool_ev.h

```diff
@@ -28,6 +28,10 @@
  * THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
  * SUCH DAMAGE.
  */
+
+/* Clang supports GCC's pragma. */
+#pragma GCC system_header
```

Comment:
Nit: It'd be nice to have a comment here explaining why we need to mark this header as system.

Reply:
Added a brief comment about the need to ignore `-Wstrict-aliasing` compiler warnings from libev.